pyproject.toml

```
@@ -1,8 +1,23 @@
 [tool.poetry]
 name = "youtrack-sdk"
-version = "0.1.0"
-description = ""
-authors = []
+version = "1.1+SNAPSHOT"
+description = "YouTrack SDK"
```

```suggestion
description = "YouTrack API SDK"
```
To make it consistent with the description of the GitHub repository?

---

I'm not sure that `YouTrack API SDK` is the correct description. For example, [boto3](https://github.com/boto/boto3) is just AWS SDK for Python, not AWS API SDK. So maybe we should update repository description instead of package description.